pkgs/development/compilers/rasm/default.nix

```diff
@@ -0,0 +1,37 @@
+{stdenv, fetchurl, unzip}:
+
+stdenv.mkDerivation rec {
+  pname = "rasm";
+  version = "0.116";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    name = "rasm_v0116_src.zip";
+    url = "http://www.cpcwiki.eu/forum/programming/rasm-z80-assembler-in-beta/?action=dlattach;attach=30015";
```

Comment:
is there a more official url, other than some download links on a forum? Would be nice if he/she had a github/gitlab, especially seeing as it's mit license anyway.

Reply:
despite how this seems, it's the official topic according to author page. It would be nice if it was opensource too but that's not.

Reply:
it's the official url according http://www.roudoudou.com/rasm/ . If i've time i would contact author but you know, most of time, such people doesn't work with git and has no opensource culture. BTW I'll try later since it's a compatriot.